README.md

```diff
@@ -36,6 +36,14 @@ where:
         -s/--source-dir: The name of the source directory.
         Default: <plugin_name>_<plugin_type>
 
+        -pd/--platform-dependencies: The platform dependencies for the Agent Plugin.
```

Comment:
Can we include the possible values? It isn't documented anywhere but the code for the moment. Is it possible that argparse can display that automagically for us? If not, consider using [click](https://click.palletsprojects.com/en/8.1.x/) instead. I'm fairly confident it generates better documentation for enums.